src/components/SectionList/SectionList.tsx

```diff
@@ -61,8 +68,8 @@ export default function SectionList({ sections }: Props) {
                                 { image, title: listItemTitle, subtitle },
                                 idx
                             ) => (
-                                <li key={idx}>
-                                    <ListItem divider button>
+                                <span key={idx}>
```

Comment:
Was this changed due to accessibility?

Reply:
When we were using SectionList instead of SettingsMenu, we kept getting <li> cannot be child of <p> warning or something similar so we changed it, but then after we switched over to SettingsMenu, we just forgot to change it back

Reply:
Okay, no worries.  I wanted to understand the reasoning behind the change and make sure it was intentional and not something left over from just trying things out.  Although with the new component it sounds unnecessary now.